model/src/main/kotlin/ProjectAnalyzerResult.kt

```diff
@@ -89,4 +89,27 @@ data class ProjectAnalyzerResult(
             }
         }
     }
+
+    fun collectPackages(filterByScopeName: String): List<CuratedPackage> {
```

Comment:
Consider renaming this function to `collectPackagesByScope(scopeName: String)`.

Reply:
done